rules-reviewed/jboss-eap/websphere/websphere.windup.xml

```diff
@@ -0,0 +1,701 @@
+<?xml version="1.0"?>
+<ruleset xmlns="http://windup.jboss.org/v1/xml" id="websphere_rules_general">
+    <rules>
+        <rule id="websphere_rules_general-7Apvvd">
+            <when>
+                <javaclass references="com.ibm.websphere.sib.api.jms.JmsTopicConnectionFactory" as="default"/>
+            </when>
+            <perform>
+                    <hint effort="3" severity="mandatory">
```

Comment:
According to the documentation, I had to put here 3. But in case of multiple cases matches, it may end up at 30+.